harness.go

```diff
@@ -31,12 +33,41 @@ func (s search) Description() grpc.ServiceDesc {
 	return _SearchService_serviceDesc
 }
 
+type account struct{}
+
+func (s account) GetAccountInfo(ctx context.Context, r *GetAccountInfoParams) (*AccountInfo, error) {
+	fmt.Println("GetAccountInfo", r)
+	return &AccountInfo{}, nil
+
+}
+
+func (s account) Description() grpc.ServiceDesc {
+	return _AccountService_serviceDesc
+}
+
+func mustLogin(in http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if true {
+			http.Redirect(w, r, "/", http.StatusTemporaryRedirect)
+		}
+		in.ServeHTTP(w, r)
+	})
+}
+
 func main() {
+
+	mux := http.NewServeMux()
+	a := account{}
 	s := search{}
 
-	mux := wrapper.Wrap(s)
+	wrapper.Handle(mux, a, mustLogin(wrapper.Wrap(a)))
+	wrapper.Handle(mux, s, wrapper.Wrap(s))
```

Comment:
I find it a bit awkward that you need to pass `s` and `wrapper.Wrap(s)` to `Handle`.